tests/files/c_source/abs.h

```diff
@@ -58,6 +58,63 @@
 #define ABS_BREMSE_50_FRAME_ID (0x560u)
 #define ABS_BREMSE_53_FRAME_ID (0x5c0u)
 
+#define ABS_BREMSE_33_FRAME_LEN (8u)
```

Comment:
I suggest `#define ABS_BREMSE_33_LENGTH (8u)` to have the same naming as the database message class.